mapie/tests/test_classification.py

```diff
@@ -1606,28 +1612,16 @@ def test_method_error_in_fit(monkeypatch: Any, method: str) -> None:
         mapie_clf.fit(X_toy, y_toy)
 
 
-@pytest.mark.parametrize("method", WRONG_METHODS)
```

Comment:
Why do we no longer perform this test?

Reply:
This test tested, in the case where the user changes the method value between fit and predict, that an error was reported.

But now, we manage non-conformity scores. Due to the logic of the refacto, this test in the code no longer makes sense because we do not modify the non-confnormity score as we want.

1) We do not do this test for MapieRegressor
2) We do not do this test if the estimator has changed between the fit and the predict